rllib/algorithms/appo/default_appo_rl_module.py

```diff
@@ -0,0 +1,54 @@
+import abc
+from typing import Any, Dict, List, Tuple
+
+from ray.rllib.algorithms.ppo.default_ppo_rl_module import DefaultPPORLModule
+from ray.rllib.algorithms.appo.appo import OLD_ACTION_DIST_LOGITS_KEY
+from ray.rllib.core.learner.utils import make_target_network
+from ray.rllib.core.models.base import ACTOR
+from ray.rllib.core.models.tf.encoder import ENCODER_OUT
+from ray.rllib.core.rl_module.apis import TargetNetworkAPI
+from ray.rllib.utils.typing import NetworkType
+
+from ray.rllib.utils.annotations import (
+    override,
+    OverrideToImplementCustomLogic_CallToSuperRecommended,
+)
+
+
+class DefaultAPPORLModule(DefaultPPORLModule, TargetNetworkAPI, abc.ABC):
```

Comment:
I like this default naming. Do we apply it to all algorithms now? That would be great!

Reply:
Yeah, exactly, that's the idea! Only DQN and SAC missing, I think ...

Reply:
DQN and the rest will be done in the [next PR](https://github.com/ray-project/ray/pull/49368) (already in review ...)